src/WebJobs.Script.Host/WebJobs.Script.Host.csproj

```diff
@@ -120,7 +120,7 @@
       <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.DocumentDB.1.3.0-beta1-10632\lib\net45\Microsoft.Azure.WebJobs.Extensions.DocumentDB.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.WebJobs.Extensions.EventGrid, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.EventGrid.1.0.0-10036\lib\net46\Microsoft.Azure.WebJobs.Extensions.EventGrid.dll</HintPath>
```

Comment:
Nuget flags 1.0.0-10036 as an invalid package version, causing all VS builds to fail on all of my dev machines. We don't have an actual product issue since apparently this doesn't repro on AppVeyor.

Confirmed with @watashiSHUN that 1.0.0 is the same bits as this version so this is safe to do. Going forward, he's going to fix this versioning scheme.